cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.ts is processed and
```

Comment:
Same with this file, is it needed?

Reply:
this file is needed to run the tests.
I remove useless imports and add a devnote to explain the goal of the file